src/lib/components/common/Button.svelte

```diff
@@ -1,8 +1,13 @@
 <script>
+	export let variant = 'primary';
 </script>
 
 <button
-	class="w-full bg-lkud-red-primary text-2xl py-2 text-lkud-yellow-primary font-monts"
+	class={`w-full ${
+		variant === 'primary'
+			? 'bg-lkud-red-primary text-2xl py-2 text-lkud-yellow-primary'
+			: ' bg-lkud-yellow-primary text-lkud-red-primary text-2xl py-2'
+	}  font-monts`}
```

Comment:
There's a better way to do this in svelte.

Put the variant specific styles in the style tag [https://svelte.dev/tutorial/styling](https://svelte.dev/tutorial/styling)

Then use the shorthand class directive [https://svelte.dev/tutorial/classes](https://svelte.dev/tutorial/classes)

```svelte
<button
    class:primary={variant == 'primary'}
>
...
</button>
```

Reply:
fixed